src/backend/optimizer/path/allpaths.c

```diff
@@ -356,9 +356,16 @@ set_plain_rel_pathlist(PlannerInfo *root, RelOptInfo *rel, RangeTblEntry *rte)
 	if (root->config->enable_seqscan)
```

Comment:
@t1mursadykov I can see that the original commit [removed](https://github.com/greenplum-db/gpdb/commit/cd055f999f87c8bd381cc3fd7a64ee94dfa46073#diff-c5a068ff48b9a5ef8e69eb30333d223479b794bc6771f8953795815bb688d5bdL349-L352) sequential, tid, index and bitmap scans from `DefaultPlannerConfig` (so we don't use `root->config->enable_XXXscan` any more). You didn't backport this change - have it been done on purpose?

Reply:
Do we really need it? As I understand it, this is just refactoring.

Reply:
But you already [replace](https://github.com/arenadata/gpdb/pull/172/files#diff-4eff9e11378a8d7655a2cd15eea18c2fa93687163c15bcdd6e4dd54818fef85fL356) `root->config->enable_seqscan` with `enable_seqscan`. If you don't want to backport modifications of `DefaultPlannerConfig`, this replacement makes no sense.